client/task-list/tasks.js

```diff
@@ -58,95 +58,79 @@ export function getAllTasks( {
 	);
 
 	const tasks = [
+		{
+			key: 'store_details',
+			title: __( 'Store details', 'woocommerce-admin' ),
+			container: null,
+			onClick: () => {
+				window.location = getAdminLink(
+					'admin.php?page=wc-admin&reset_profiler=1'
+				);
+			},
```

Comment:
Perhaps good for a follow up: We should use `<Link />` component with these list items. This `onClick` causes a page refresh where one isn't required. Clicking "Store details" should not refresh the page. Secondarily, it removes the "preview" address in the lower right hand side which is useful.

Reply:
I'd definitely like to tackle this in a follow up - at a guess, this is likely to be more involved than changing to a link component since we're potentially going to need to reset the "completed" flag for the task list (and think through the impact to tracking of that completion).